src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -50,10 +42,6 @@ export default {
             options: Object.keys(MultiSelectType),
             control: { type: 'select' },
         },
-        emphasis: {
```

Comment:
This should stay and change the Trigger Style. We should be able to control that in Storybook. Also it would be nice to add a Story for weak Trigger.